src/main/java/org/nixos/idea/imports/NixFilePathReferenceContributor.kt

```diff
@@ -0,0 +1,63 @@
+package org.nixos.idea.imports
+
+import com.intellij.codeInsight.lookup.LookupElement
+import com.intellij.openapi.util.TextRange
+import com.intellij.openapi.util.io.FileUtil
+import com.intellij.openapi.vfs.LocalFileSystem
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.openapi.vfs.VirtualFileSystem
+import com.intellij.patterns.PlatformPatterns
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiManager
+import com.intellij.psi.PsiReference
+import com.intellij.psi.PsiReferenceBase
+import com.intellij.psi.PsiReferenceContributor
+import com.intellij.psi.PsiReferenceProvider
+import com.intellij.psi.PsiReferenceRegistrar
+import com.intellij.util.ProcessingContext
+import org.nixos.idea.psi.impl.NixExprStdPathMixin
+
+class NixFilePathReferenceContributor: PsiReferenceContributor() {
```

Comment:
issue (non-blocking): Would be nice to have some test to cover this functionality. However, not sure how a test could look like. I have some tests regarding references in `SymbolNavigationTest`, but their setup is probably far too complex for your use case, and it also doesn't support asserting for references to files.

It is not a hard requirement before I merge the PR, but it would be nice.

Reply:
You mean as a live call? I somehow didn't expect that in the context of a pull request on GitHub. 😄 Unfortunately, it is difficult for me to plan specific times before the start of January. Not sure if I want to wait for that. I suppose you are not on the 38C3 in Hamburg (Germany), are you?^^

Reply:
yeah! a live call! i prefer it as we can communicate a bit faster. seconds rtt instead of minutes / hours / days.

no plans to go to this 38C3 thing, seems interesting though maybe that could change, will see whether it makes sense.

what if we do this, let's merge the PR if you're fine with the lack of tests, let's get a time on the books (probably 30m) for early Jan and if i make it to 38C3 (unlikely though) we can meetup and solve it in real life? how does jan 1 sound (probably like after 5pm germany time?)